projects/ccf-rui/src/app/shared/components/labeled-side-toggle/labeled-side-toggle.component.theme.scss

```diff
@@ -0,0 +1,18 @@
+@use '~@angular/material/theming' as *;
+
+@mixin theme($theme) {
+  $foreground: map-get($theme, foreground);
+
+  .gender-slider, .side-slider {
```

Comment:
Again this NEEDS to be wrapped inside a specific selector i.e. `.ccf-labeled-slide-toggle { ... }`
CHECK other components (block-size-input, rotation-slider, etc.) for how it is done!

Reply:
Isn't this already selecting the two sliders specifically? Sorry if I'm misunderstanding.

Reply:
Yes, but it has the potential of affecting other components. Ex.
Component A theme file:
```
.label {
  color: blue;
}
```

Component B HTML:
```
<span class="label">This text becomes blue because of A</span>
```

Not a problem if component A does this in its theme file:
```
.ccf-component-a { // This now only affects things in component A
  .label {
    color: blue;
  }
}
```